README.md

```diff
@@ -0,0 +1,180 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="erd.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h1><b>Recipe App</b></h3>
+  <p>(Project ERD)</p>
+
```

Comment:
-   [ ] [OPTIONAL] Hi team, starting the server resulted in an error because the asset "tailwind.css" is not present in the asset pipeline, causing an ActionView::Template::Error.

<img width="660" alt="Screenshot 2023-07-14 at 21 05 12" src="https://github.com/peterdtitan/food-recipe-app/assets/58692590/9ae97980-1afb-4c06-bc15-fc73dfcacd54">

So, I think it will be nice to also add the command to precompile the assets since you used Tailwind for styles.

``` ruby
bundle exec rake assets:precompile
```

Reply:
Hello @Whoistolu Thanks 😍🎉for your valuable suggestion. We'll do it.